python-package/lightgbm/plotting.py

```diff
@@ -369,8 +402,15 @@ def plot_metric(booster, metric=None, dataset_names=None,
     return ax
 
 
-def _to_graphviz(tree_info, show_info, feature_names, precision=3,
-                 orientation='horizontal', constraints=None, **kwargs):
+def _to_graphviz(
+    tree_info, 
```

Comment:
```suggestion
    tree_info: List[Dict[str, Any]], 
```

I know this argument is not currently documented, but I found its expected type by looking at `lightgbm`'s tests for some code that eventually ends up calling `_to_graphviz()`.

Based on

https://github.com/microsoft/LightGBM/blob/a8fae81fc64a52d1f0b3fae5019870d7609d64c2/tests/python_package_test/test_plotting.py#L134

I ran the following to figure out the shape of this

```python
from sklearn.datasets import load_breast_cancer
from sklearn.model_selection import train_test_split

import lightgbm as lgb

X_train, _, y_train, _ = train_test_split(
    *load_breast_cancer(return_X_y=True),
    test_size=0.1,
    random_state=1
)

constraints = [-1, 1] * int(X_train.shape[1] / 2)
gbm = lgb.LGBMClassifier(n_estimators=10, num_leaves=3, silent=True, monotone_constraints=constraints)
gbm.fit(X_train, y_train, verbose=False)

# figure out shape of tree_info
model = gbm.booster_.dump_model()
tree_infos = model['tree_info']
tree_infos
```